plugins/flytekit-sqlalchemy/flytekitplugins/sqlalchemy/task.py

```diff
@@ -61,7 +61,7 @@ def __init__(
         super().__init__(
             name=name,
             task_config=task_config,
-            container_image="ghcr.io/flyteorg/flytekit:sqlalchemy-b7ccc96d46a239f12e0b65ad749d1e11d5d20f46",
+            container_image="ghcr.io/flyteorg/flytekit:sqlalchemy-6deb81af74ce8f3768553c188ab35660c717420a",
```

Comment:
where is this sha from?  did you make it manually?

Reply:
It has been auto-generated.
https://github.com/flyteorg/flytekit/blob/master/.github/workflows/sqlachemy_publish.yml